app/src/main/java/com/eveningoutpost/dexdrip/utils/BgToSpeech.java

```diff
@@ -97,6 +96,7 @@ private static String calculateText(double value, String delta_name) {
 
         final boolean doMgdl = (Pref.getString("units", "mgdl").equals("mgdl"));
         final boolean bg_to_speech_repeat_twice = (Pref.getBooleanDefaultFalse("bg_to_speech_repeat_twice"));
+        final boolean bg_to_speech_trend = (Pref.getBooleanDefaultFalse("bg_to_speech_trend"));
```

Comment:
Isn't this conditional already handled in `realSpeakNow()` ?

Reply:
Yes it is, but I see in my tests that on any reason, if I deselect the option of speaking trend, that a null value will be put into the string, also if the test condition did not allow that. So I made that as an double safety reason.